addons/mrp_subcontracting_dropshipping/tests/test_sale_dropshipping.py

```diff
@@ -326,6 +326,6 @@ def test_kit_dropshipped_change_qty_SO(self):
         })
         so.action_confirm()
 
-        user_admin = self.env['res.users'].search([('login', '=', 'admin')])
+        user_admin = self.env['res.users'].search([('name', '=', 'Odoobot')])
```

Comment:
Set the email up correctly instead of using Odoobot.

Like it was done here:
https://github.com/odoo/odoo/blob/a50e6ea418ca647ad3256ff4a24800834876d818/addons/sale_mrp/tests/test_sale_mrp_kit_bom.py#L14-L16

except you can do it directly in the test, it's not needed in the setup